src/NearOrg/Notifications/Notification.jsx

```diff
@@ -136,12 +136,13 @@ let { blockHeight, accountId, manageNotification, permission } = props;
 let postUrl = "";
 
 // Construct DevGov postUrl
+// TODO: refactor this method
 function buildPostUrl(widgetName, linkProps) {
   linkProps = { ...linkProps };
 
   const nearDevGovGigsWidgetsAccountId =
     props.nearDevGovGigsWidgetsAccountId ||
-    (context.widgetSrc ?? "devgovgigs.near").split("/", 1)[0];
+    "devgovgigs.near";
```

Comment:
In our case there is no need to check `context.widgetSrc` because all the DevHub components (like `Post`) are stored in `devgovgigs.near`. I'd like to refactor this method but after we confirm this fix works on production